easybuild/easyconfigs/g/GROMOS++/GROMOS++-1.0.2211-goolf-1.5.14-openmp.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Benjamin P. Roberts
+# New Zealand eScience Infrastructure
+# The University of Auckland, Auckland, New Zealand 
+
+easyblock = 'ConfigureMake'
+
+name = 'GROMOS++'
+version = '1.0.2211'
+versionsuffix = '-openmp'
+
+homepage = 'http://www.gromos.net/'
+description = """ GROMOS™ is an acronym of the GROningen MOlecular Simulation
+computer program package, which has been developed since 1978 for the dynamic
+modelling of (bio)molecules, until 1990 at the University of Groningen, The
+Netherlands, and since then at the ETH, the Swiss Federal Institute of
+Technology, in Zürich, Switzerland. """ 
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+toolchainopts = {'openmp': True, 'usempi': False}
+
+sources = [SOURCELOWER_TAR_BZ2]
+
+dependencies = [('GSL', '1.16')]
+
+preconfigopts = './Config.sh &&'
+configopts = '--enable-openmp'
+
+sanity_check_paths = {
+    'files': ["bin/pdb2g96", "bin/gromacs2gromos", "bin/structure_factor",
+              "lib/gromosplugin.a", "lib/libgromos.a"],
+    'dirs': ["include", "share/gromos++"],
+}
+
+group = 'gromos'
```

Comment:
testing by Jenkins fails because this group spec is here

easyconfigs should never include a group spec, since that's a site-specific thing

either remove it, or comment it out please